app/styles/_vars.scss

```diff
@@ -5,6 +5,19 @@ $transition-timing: 250ms;
 $transition-timing-queue: 150ms;
 $spacing: 1.5rem;
 
+//COLORS
+$c-gray--dark: #333333;
+$c-denim-blue: #00518e;
+
+//FONTS 
+$font-heading: "Playfair Display";
+$font-body: "OpenSans-Regular", sans-serif; 
+$font-body-bold: "OpenSans-Bold", sans-serif;
+
```

Comment:
I think you could put these inside the _onboarding.scss file since they only pertain to this feature, and call them something different. This makes it sound like the whole site going to use these fonts and that's not the case right now